expat/PSPBUILD

```diff
@@ -9,37 +9,24 @@ license=('MIT')
 depends=()
```

Comment:
The pkgrel for this package should probably be increased, depending on if the version number is changed or not.

Reply:
Done